SmartDeviceLink/SDLPutFile.m

```diff
@@ -29,6 +31,19 @@ - (instancetype)initWithFileName:(NSString *)fileName fileType:(SDLFileType)file
     return self;
 }
 
+- (instancetype)initWithFileName:(NSString *)fileName fileType:(SDLFileType)fileType persistentFile:(BOOL)persistentFile systemFile:(BOOL)systemFile offset:(UInt32)offset length:(UInt32)length crc:(UInt64)crc {
```

Comment:
These should be in the same order as the `.h`, there are a few initializers that don't exist in the `.h` at all. Those don't seem necessary and can be removed.

Reply:
Fixed. Removed the unused convenience inits and updated the unit test cases.